### PR TITLE
Preempted to backendStatus in existing metadata. Develop edition

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: database/migration/src/main/scala/cromwell/database/migration/custom/PreemptedIsABackendStatus.scala [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  override def migrateBatchRow(readRow: ResultSet, migrateStatements: List[PreparedStatement]): Int = {
+    val rowId = readRow.getInt(primaryKeyColumn)
+    val workflowId = readRow.getString(workflowIdColumn)
+    val callFqn = readRow.getString(callFqnColumn)
+    val jobScatterIndex = readRow.getObject(jobScatterIndexColumn)
+    val retryAttempt = readRow.getObject(retryAttemptColumn)
+    val metadataTimestamp = readRow.getObject(timestampColumn)
+
+    // First, update the executionStatus to be
```
Comment (geoffjentry) [inline database/migration/src/main/scala/cromwell/database/migration/custom/PreemptedIsABackendStatus.scala]: to be ...... ?

File: database/migration/src/main/scala/cromwell/database/migration/custom/PreemptedIsABackendStatus.scala [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    val insertStatement = migrateStatements(1)
+    insertStatement.setString(1, workflowId)
+    insertStatement.setString(2, callFqn)
+    insertStatement.setObject(3, jobScatterIndex)
+    insertStatement.setObject(4, retryAttempt)
+    insertStatement.setObject(5, metadataTimestamp)
+    insertStatement.addBatch()
+
+    // We added two queries here, so that's what we return:
+    2
```
Comment (geoffjentry) [inline database/migration/src/main/scala/cromwell/database/migration/custom/PreemptedIsABackendStatus.scala]: this made me chuckle for reasons i can't verbalize.

**Comment by geoffjentry** [comment]
👍 

How did you test this?

**Comment by mcovarr** [comment]
:+1:

**Comment by cjllanwarne** [comment]
@geoffjentry:
- Created a database entry with a `Preempted` executionStatus, to see that the problem was occurring.
- Ran the migration and checked that the metadata was indeed now visible

**Pull request merged**